### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abhatia2107**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ To get started:
 - Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
   - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
 
-Then follow the on-screen instructions to set up your repo.
+Then follow the on-screen instructions to set up your repository.
```
Comment (Ishika11) [inline README.md:12]: revert the change

Comment (abhatia2107) [reply README.md:12]: Not doing it.
